"use strict";

const geneResponse = {
  gene: {
    geneInfo: {
      id: 'RDBECOLIGN03613',
      name: 'lon',
      leftEndPosition: 458888,
      rightEndPosition: 461242,
      strand: 'forward',
      sequence: 'ATGAATCCTGAGCGTTCTGAACGCATTGAAATCCCCGTATTGCCGCTGCGCGATGTGGTGGTTTATCCGCACATGGTCATCCCCTTATTTGTCGGGCGGGAAAAATCTATCCGTTGTCTGGAAGCGGCGATGGACCATGATAAAAAAATTATGCTGGTCGCGCAGAAAGAAGCTTCAACGGATGAGCCGGGTGTAAACGATCTTTTCACCGTCGGGACCGTGGCCTCTATATTGCAGATGCTGAAACTGCCTGACGGCACCGTCAAAGTGCTGGTCGAGGGGTTACAGCGCGCGCGTATTTCTGCGCTCTCTGACAATGGCGAACACTTTTCTGCGAAGGCGGAGTATCTGGAGTCGCCGACCATTGATGAGCGGGAACAGGAAGTGCTGGTGCGTACTGCAATCAGCCAGTTCGAAGGCTACATCAAGCTGAACAAAAAAATCCCACCAGAAGTGCTGACGTCGCTGAATAGCATCGACGATCCGGCGCGTCTGGCGGATACCATTGCTGCACATATGCCGCTGAAACTGGCTGACAAACAGTCTGTTCTGGAGATGTCCGACGTTAACGAACGTCTGGAATATCTGATGGCAATGATGGAATCGGAAATCGATCTGCTGCAGGTTGAGAAACGCATTCGCAACCGCGTTAAAAAGCAGATGGAGAAATCCCAGCGTGAGTACTATCTGAACGAGCAAATGAAAGCTATTCAGAAAGAACTCGGTGAAATGGACGACGCGCCGGACGAAAACGAAGCCCTGAAGCGCAAAATCGACGCGGCGAAGATGCCGAAAGAGGCAAAAGAGAAAGCGGAAGCAGAGTTGCAGAAGCTGAAAATGATGTCTCCGATGTCGGCAGAAGCGACCGTAGTGCGTGGTTATATCGACTGGATGGTACAGGTGCCGTGGAATGCGCGTAGCAAGGTCAAAAAAGACCTGCGTCAGGCGCAGGAAATCCTTGATACCGACCATTATGGTCTGGAGCGCGTGAAAGATCGAATCCTTGAGTATCTTGCGGTTCAAAGCCGTGTCAACAAAATCAAGGGACCGATCCTCTGCCTGGTAGGGCCGCCGGGGGTAGGTAAAACCTCTCTTGGTCAGTCCATTGCCAAAGCCACCGGGCGTAAATATGTCCGTATGGCGCTGGGCGGCGTGCGTGATGAAGCGGAAATCCGTGGTCACCGCCGTACTTACATCGGTTCTATGCCGGGTAAACTGATCCAGAAAATGGCGAAAGTGGGCGTGAAAAACCCGCTGTTCCTGCTCGATGAGATCGACAAAATGTCTTCTGACATGCGTGGCGATCCGGCCTCTGCACTGCTTGAAGTGCTGGATCCAGAGCAGAACGTAGCGTTCAGCGACCACTACCTGGAAGTGGATTACGATCTCAGCGACGTGATGTTTGTCGCGACGTCGAACTCCATGAACATTCCGGCACCGCTGCTGGATCGTATGGAAGTGATTCGCCTCTCCGGTTATACCGAAGATGAAAAACTGAACATCGCCAAACGTCACCTGCTGCCGAAGCAGATTGAACGTAATGCACTGAAAAAAGGTGAGCTGACCGTCGACGATAGCGCCATTATCGGCATTATTCGTTACTACACCCGTGAGGCGGGCGTGCGTGGTCTGGAGCGTGAAATCTCCAAACTGTGTCGCAAAGCGGTTAAGCAGTTACTGCTCGATAAGTCATTAAAACATATCGAAATTAACGGCGATAACCTGCATGACTATCTCGGTGTTCAGCGTTTCGACTATGGTCGCGCGGATAACGAAAACCGTGTCGGTCAGGTAACCGGTCTGGCGTGGACGGAAGTGGGCGGTGACTTGCTGACCATTGAAACCGCATGTGTTCCGGGTAAAGGCAAACTGACCTATACCGGTTCGCTCGGCGAAGTGATGCAGGAGTCCATTCAGGCGGCGTTAACGGTGGTTCGTGCGCGTGCGGAAAAACTGGGGATCAACCCTGATTTTTACGAAAAACGTGACATCCACGTCCACGTACCGGAAGGTGCGACGCCGAAAGATGGTCCGAGTGCCGGTATTGCTATGTGCACCGCGCTGGTTTCTTGCCTGACCGGTAACCCGGTTCGTGCCGATGTGGCAATGACCGGTGAGATCACTCTGCGTGGTCAGGTACTGCCGATCGGTGGTTTGAAAGAAAAACTCCTGGCAGCGCATCGCGGCGGGATTAAAACAGTGCTAATTCCGTTCGAAAATAAACGCGATCTGGAAGAGATTCCTGACAACGTAATTGCCGATCTGGACATTCATCCTGTGAAGCGCATTGAGGAAGTTCTGACTCTGGCGCTGCAAAATGAACCGTCTGGTATGCAGGTTGTGACTGCAAAATAG',
      gcContent: 52.31,
      centisomePosition: 9.886308,
      type: 'gene',
      synonyms: ['capR', 'deg', 'dir', 'lopA', 'muc'],
      terms: {
        multifun: [['1 - metabolism', '1.6 - biosynthesis of macromolecules (cellular constituents)', '1.6.13 - fimbri, pili'], ['2 - information transfer', '2.1 - DNA related', '2.1.3 - DNA recombination'], ['3 - regulation', '3.1 - type of regulation', '3.1.1 - DNA structure level'], ['3 - regulation', ' 3.3 - genetic unit regulated', ' 3.3.1 - operon'], ['6 - cell structure', '6.5 - pilus']],
        geneOntology: {
          cellularComponent: [{
            id: 'GO:0005829',
            name: 'cytosol'
          }],
          molecularFunction: [{
            id: 'GO:0003677',
            name: 'DNA binding'
          }],
          biologicalProcess: [{
            id: 'GO:0006310',
            name: 'DNA recombination'
          }, {
            id: 'GO:0006351',
            name: 'transcription, DNA-templated'
          }, {
            id: 'GO:0006355',
            name: 'regulation of transcription, DNA-templated'
          }, {
            id: 'GO:0006974',
            name: 'cellular response to DNA damage stimulus'
          }, {
            id: 'GO:0015074',
            name: 'DNA integration'
          }]
        }
      },
      externalCrossReferences: [{
        id: '511145.b4312',
        name: 'String',
        url: 'http://string-db.org/newstring_cgi/show_network_section.pl?identifier=511145.b4312'
      }, {
        id: 'ABE-0014141',
        name: 'ASAP',
        url: 'http://asap.ahabs.wisc.edu/asap/feature_info.php?FeatureID=ABE-0014141'
      }, {
        id: 'b4312',
        name: 'OU-Microarray',
        url: 'https://genexpdb.okstate.edu/databases/genexpdb/?query=b4312'
      }, {
        id: '18355',
        name: 'Coli Genetic Stock Center',
        url: 'http://cgsc.biology.yale.edu/Site.php?ID=18355'
      }, {
        id: 'fimB',
        name: 'PortEco',
        url: 'http://porteco.org/AjaxSearch.jsp?searchString=fimB'
      }, {
        id: 'EB0305',
        name: 'EchoBASE',
        url: 'http://www.york.ac.uk/res/thomas/Gene.cfm?recordID=EB0305'
      }, {
        id: 'b4312',
        name: 'RegulonDB',
        url: 'http://regulondb.ccg.unam.mx/search?term=b4312&organism=ECK12&type=All'
      }],
      evidenceReferences: [{
        evidenceName: 'Evidence Computational',
        evidenceCode: 'EV-COMP',
        evidenceType: 'Strong',
        referenceId: 9108285,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/9108285',
        referenceCitation: 'Dove, S., Smith, S. & Dorman, C. Control of Escherichia coli type 1 fimbrial gene expression in stationary phase: a negative role for RpoS. Mol Gen Genet 254, 13–20 (1997) doi:10.1007/s004380050385'
      }]
    },
    products: [{
      regulatorId: 'RDBECOLIRG00001',
      name: 'Lon protease',
      sequence: 'MNPERSERIEIPVLPLRDVVVYPHMVIPLFVGREKSIRCLEAAMDHDKKIMLVAQKEASTDEPGVNDLFTVGTVASILQMLKLPDGTVKVLVEGLQRARISALSDNGEHFSAKAEYLESPTIDEREQEVLVRTAISQFEGYIKLNKKIPPEVLTSLNSIDDPARLADTIAAHMPLKLADKQSVLEMSDVNERLEYLMAMMESEIDLLQVEKRIRNRVKKQMEKSQREYYLNEQMKAIQKELGEMDDAPDENEALKRKIDAAKMPKEAKEKAEAELQKLKMMSPMSAEATVVRGYIDWMVQVPWNARSKVKKDLRQAQEILDTDHYGLERVKDRILEYLAVQSRVNKIKGPILCLVGPPGVGKTSLGQSIAKATGRKYVRMALGGVRDEAEIRGHRRTYIGSMPGKLIQKMAKVGVKNPLFLLDEIDKMSSDMRGDPASALLEVLDPEQNVAFSDHYLEVDYDLSDVMFVATSNSMNIPAPLLDRMEVIRLSGYTEDEKLNIAKRHLLPKQIERNALKKGELTVDDSAIIGIIRYYTREAGVRGLEREISKLCRKAVKQLLLDKSLKHIEINGDNLHDYLGVQRFDYGRADNENRVGQVTGLAWTEVGGDLLTIETACVPGKGKLTYTGSLGEVMQESIQAALTVVRARAEKLGINPDFYEKRDIHVHVPEGATPKDGPSAGIAMCTALVSCLTGNPVRADVAMTGEITLRGQVLPIGGLKEKLLAAHRGGIKTVLIPFENKRDLEEIPDNVIADLDIHPVKRIEEVLTLALQNEPSGMQVVTAK',
      molecularWeight: 87.438,
      isoelectricPoint: 6.313,
      cellularLocation: 'cytosol',
      synonyms: ['Muc', 'Dir', 'Deg', 'LopA', 'CapR', 'Lon', 'DNA-binding, ATP-dependent protease La'],
      motifs: [{
        sequence: 'M',
        type: '|Cleavage-of-Initial-Methionine|',
        note: 'UniProt: Removed.'
      }, {
        leftEndPosition: 570,
        rightEndPosition: 772,
        sequence: 'INGDNLHDYLGVQRFDYGRADNENRVGQVTGLAWTEVGGDLLTIETACVPGKGKLTYTGSLGEVMQESIQAALTVVRARAEKLGINPDFYEKRDIHVHVPEGATPKDGPSAGIAMCTALVSCLTGNPVRADVAMTGEITLRGQVLPIGGLKEKLLAAHRGGIKTVLIPFENKRDLEEIPDNVIADLDIHPVKRIEEVLTLALQ',
        type: '|Pfam-Features|'
      }, {
        leftEndPosition: 264,
        rightEndPosition: 317,
        sequence: 'PKEAKEKAEAELQKLKMMSPMSAEATVVRGYIDWMVQVPWNARSKVKKDLRQAQ',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 5; AAA24078).'
      }, {
        leftEndPosition: 356,
        rightEndPosition: 363,
        sequence: 'GPPGVGKT',
        type: '|Nucleotide-Phosphate-Binding-Regions|',
        note: 'UniProt: ATP.'
      }, {
        leftEndPosition: 240,
        rightEndPosition: 252,
        sequence: 'ELGEMDDAPDENE',
        type: '|Protein-Segments|',
        note: 'UniProt: Asp/Glu-rich (acidic); Sequence Annotation Type: compositionally biased region.'
      }, {
        sequence: 'M',
        type: '|Active-Sites|'
      }, {
        leftEndPosition: 10,
        rightEndPosition: 201,
        sequence: 'EIPVLPLRDVVVYPHMVIPLFVGREKSIRCLEAAMDHDKKIMLVAQKEASTDEPGVNDLFTVGTVASILQMLKLPDGTVKVLVEGLQRARISALSDNGEHFSAKAEYLESPTIDEREQEVLVRTAISQFEGYIKLNKKIPPEVLTSLNSIDDPARLADTIAAHMPLKLADKQSVLEMSDVNERLEYLMAMME',
        type: '|Pfam-Features|'
      }, {
        leftEndPosition: 679,
        rightEndPosition: 679,
        sequence: 'S',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity.'
      }, {
        leftEndPosition: 772,
        rightEndPosition: 772,
        sequence: 'Q',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 4; AAA16837).'
      }, {
        leftEndPosition: 307,
        rightEndPosition: 307,
        sequence: 'S',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 4; AAA16837).'
      }, {
        leftEndPosition: 2,
        rightEndPosition: 784,
        sequence: 'NPERSERIEIPVLPLRDVVVYPHMVIPLFVGREKSIRCLEAAMDHDKKIMLVAQKEASTDEPGVNDLFTVGTVASILQMLKLPDGTVKVLVEGLQRARISALSDNGEHFSAKAEYLESPTIDEREQEVLVRTAISQFEGYIKLNKKIPPEVLTSLNSIDDPARLADTIAAHMPLKLADKQSVLEMSDVNERLEYLMAMMESEIDLLQVEKRIRNRVKKQMEKSQREYYLNEQMKAIQKELGEMDDAPDENEALKRKIDAAKMPKEAKEKAEAELQKLKMMSPMSAEATVVRGYIDWMVQVPWNARSKVKKDLRQAQEILDTDHYGLERVKDRILEYLAVQSRVNKIKGPILCLVGPPGVGKTSLGQSIAKATGRKYVRMALGGVRDEAEIRGHRRTYIGSMPGKLIQKMAKVGVKNPLFLLDEIDKMSSDMRGDPASALLEVLDPEQNVAFSDHYLEVDYDLSDVMFVATSNSMNIPAPLLDRMEVIRLSGYTEDEKLNIAKRHLLPKQIERNALKKGELTVDDSAIIGIIRYYTREAGVRGLEREISKLCRKAVKQLLLDKSLKHIEINGDNLHDYLGVQRFDYGRADNENRVGQVTGLAWTEVGGDLLTIETACVPGKGKLTYTGSLGEVMQESIQAALTVVRARAEKLGINPDFYEKRDIHVHVPEGATPKDGPSAGIAMCTALVSCLTGNPVRADVAMTGEITLRGQVLPIGGLKEKLLAAHRGGIKTVLIPFENKRDLEEIPDNVIADLDIHPVKRIEEVLTLALQNEPSGMQVVTAK',
        type: '|Chains|',
        note: 'UniProt: Lon protease.'
      }, {
        leftEndPosition: 255,
        rightEndPosition: 270,
        sequence: 'KRKIDAAKMPKEAKEK',
        type: '|Protein-Segments|',
        note: 'UniProt: Arg/Lys-rich (basic); Sequence Annotation Type: compositionally biased region.'
      }, {
        leftEndPosition: 11,
        rightEndPosition: 202,
        sequence: 'IPVLPLRDVVVYPHMVIPLFVGREKSIRCLEAAMDHDKKIMLVAQKEASTDEPGVNDLFTVGTVASILQMLKLPDGTVKVLVEGLQRARISALSDNGEHFSAKAEYLESPTIDEREQEVLVRTAISQFEGYIKLNKKIPPEVLTSLNSIDDPARLADTIAAHMPLKLADKQSVLEMSDVNERLEYLMAMMES',
        type: '|Conserved-Regions|',
        note: 'UniProt: Lon N-terminal.'
      }, {
        sequence: 'M',
        type: '|Active-Sites|'
      }, {
        leftEndPosition: 362,
        rightEndPosition: 362,
        sequence: 'K',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity and ATP-binding. No increased persister cell formation.'
      }, {
        leftEndPosition: 665,
        rightEndPosition: 665,
        sequence: 'H',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity.'
      }, {
        leftEndPosition: 743,
        rightEndPosition: 743,
        sequence: 'D',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: No effect.'
      }, {
        leftEndPosition: 352,
        rightEndPosition: 489,
        sequence: 'LCLVGPPGVGKTSLGQSIAKATGRKYVRMALGGVRDEAEIRGHRRTYIGSMPGKLIQKMAKVGVKNPLFLLDEIDKMSSDMRGDPASALLEVLDPEQNVAFSDHYLEVDYDLSDVMFVATSNSMNIPAPLLDRMEVIR',
        type: '|Pfam-Features|'
      }, {
        leftEndPosition: 211,
        rightEndPosition: 219,
        sequence: 'KRIRNRVKK',
        type: '|Protein-Segments|',
        note: 'UniProt: Arg/Lys-rich (basic); Sequence Annotation Type: compositionally biased region.'
      }, {
        leftEndPosition: 676,
        rightEndPosition: 676,
        sequence: 'D',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity.'
      }, {
        leftEndPosition: 779,
        rightEndPosition: 784,
        sequence: 'QVVTAK',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 2; AAA24079).'
      }, {
        leftEndPosition: 539,
        rightEndPosition: 563,
        sequence: 'AGVRGLEREISKLCRKAVKQLLLDK',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 5; AAA24078).'
      }, {
        leftEndPosition: 273,
        rightEndPosition: 273,
        sequence: 'A',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 4; AAA16837).'
      }, {
        leftEndPosition: 592,
        rightEndPosition: 773,
        sequence: 'ENRVGQVTGLAWTEVGGDLLTIETACVPGKGKLTYTGSLGEVMQESIQAALTVVRARAEKLGINPDFYEKRDIHVHVPEGATPKDGPSAGIAMCTALVSCLTGNPVRADVAMTGEITLRGQVLPIGGLKEKLLAAHRGGIKTVLIPFENKRDLEEIPDNVIADLDIHPVKRIEEVLTLALQN',
        type: '|Conserved-Regions|',
        note: 'UniProt: Lon proteolytic.'
      }, {
        leftEndPosition: 362,
        rightEndPosition: 362,
        sequence: 'K',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity and ATP-binding.'
      }, {
        leftEndPosition: 667,
        rightEndPosition: 667,
        sequence: 'H',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity.'
      }, {
        leftEndPosition: 679,
        rightEndPosition: 679,
        sequence: 'S',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity. No increased persister cell formation.'
      }],
      externalCrossReferences: [{
        id: 'PRO_000022617',
        name: 'Protein Ontology',
        url: 'http://pir.georgetown.edu/cgi-bin/pro/entry_pro?id=PRO_000022617'
      }, {
        id: 'NP_418732',
        name: 'NCBI Reference Sequences Database',
        url: 'http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=NP_418732'
      }, {
        id: 'DIP-47842N',
        name: 'Database of Interacting Proteins',
        url: 'http://dip.doe-mbi.ucla.edu/dip/Browse.cgi?ID=DIP-47842N'
      }, {
        id: 'PF00589',
        name: 'Pfam',
        url: 'http://pfam.xfam.org/family/PF00589'
      }, {
        id: 'P0ADH5',
        name: 'Pride',
        url: ' http://www.ebi.ac.uk/pride/searchSummary.do?queryTypeSelected=identification%20accession%20number&identificationAccessionNumber=P0ADH5'
      }, {
        id: 'P0ADH5',
        name: 'MODBASE',
        url: 'http://modbase.compbio.ucsf.edu/modbase-cgi/model_search.cgi?searchkw=name&kword=P0ADH5'
      }, {
        id: 'IPR011010',
        name: 'InterPro',
        url: 'http://www.ebi.ac.uk/interpro/entry/IPR011010'
      }, {
        id: 'MINT-1221137',
        name: 'Mint',
        url: 'http://mint.bio.uniroma2.it/mint/search/search.do?queryType=protein&interactorAc=MINT-1221137'
      }, {
        id: 'P0ADH5',
        name: 'SWISS-MODEL',
        url: 'http://swissmodel.expasy.org/repository/smr.php?sptr_ac=P0ADH5'
      }, {
        id: 'P0ADH5',
        name: 'UniProt',
        url: 'https://www.uniprot.org/uniprot/P0ADH5'
      }, {
        id: 'b4312',
        name: 'EcoliWiki',
        url: 'http://ecoliwiki.net/colipedia/index.php/b4312'
      }, {
        id: 'P0ADH5',
        name: 'Protein Model Portal',
        url: 'http://www.proteinmodelportal.org/query/uniprot/P0ADH5'
      }],
      evidenceReferences: [{
        evidenceName: 'Assay of unpurified protein ',
        evidenceCode: 'AUP',
        evidenceType: 'Strong',
        referenceId: 8878036,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/8878036',
        referenceCitation: 'Gally, D.L., Leathart, J. and Blomfield, I.C. (1996), Interaction of FimB and FimE with the fim switch that controls the phase variation of type 1 fimbriae in Escherichia coli K‐12. Molecular Microbiology, 21: 725-738. doi:10.1046/j.1365-2958.1996.311388.x'
      }, {
        referenceId: 10648536,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/10648536',
        referenceCitation: 'Antigen 43 and Type 1 Fimbriae Determine Colony Morphology of Escherichia coli K-12 Henrik Hasman, Mark A. Schembri, Per Klemm Journal of Bacteriology Feb 2000, 182 (4) 1089-1095; DOI: 10.1128/JB.182.4.1089-1095.2000'
      }, {
        referenceId: 11881836,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/11881836',
        referenceCitation: 'Denise M. Wolf and Adam P. Arkin.OMICS: A Journal of Integrative Biology.Jan 2002.91-114.'
      }, {
        referenceId: 10781567,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/10781567',
        referenceCitation: 'Interaction of the FimB Integrase with thefimS Invertible DNA Element in Escherichia coli In Vivo and In Vitro Lesley S. Burns, Stephen G. J. Smith, Charles J. Dorman Journal of Bacteriology May 2000, 182 (10) 2953-2959; DOI: 10.1128/JB.182.10.2953-2959.2000'
      }, {
        referenceId: 16988236,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/16988236',
        referenceCitation: 'Teng CH, Xie Y, Shin S, et al. Effects of ompA deletion on expression of type 1 fimbriae in Escherichia coli K1 strain RS218 and on the association of E. coli with human brain microvascular endothelial cells. Infect Immun. 2006;74(10):5609–5616. doi:10.1128/IAI.00321-06'
      }]
    }],
    shineDalgarno: [{
      distanceToGene: -11,
      leftEndPosition: 4540943,
      rightEndPosition: 4540948,
      sequence: 'tctgtaattaTAAGGGaaaaacgATG',
      note: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
    }],
    regulation: {
      operon: {
        id: 'RDBECOLIOP00001',
        name: 'fimB',
        arrangement: [{
          regulator: {
            id: 'RDBECOLIRG00002',
            name: 'NagC',
            type: 'activator'
          },
          promoter: {
            id: 'RDBECOLIPM00001',
            name: 'fimBp1'
          },
          transcriptionUnit: {
            id: 'RDBECOLITU00001',
            name: 'fimB'
          }
        }, {
          regulator: {
            id: 'RDBECOLIRG00003',
            name: 'IHF',
            type: 'activator'
          },
          promoter: {
            id: 'RDBECOLIPM00002',
            name: 'fimBp2'
          },
          transcriptionUnit: {
            id: 'RDBECOLITU00002',
            name: 'fimB'
          }
        }, {
          regulator: {
            id: 'RDBECOLIRG00004',
            name: 'H-NS',
            type: 'repressor'
          },
          promoter: {
            id: 'RDBECOLIPM00003',
            name: 'fimBp3'
          },
          transcriptionUnit: {
            id: 'RDBECOLITU00003',
            name: 'fimB'
          }
        }]
      },
      regulators: [{
        id: 'RDBECOLIRG00002',
        name: 'NagC',
        type: 'activator'
      }, {
        id: 'RDBECOLIRG00003',
        name: 'IHF',
        type: 'activator'
      }, {
        id: 'RDBECOLIRG00004',
        name: 'H-NS',
        type: 'repressor'
      }],
      context: [{
        type: 'promoter',
        name: 'TSS_5131',
        leftEndPosition: 4540686,
        strand: 'forward',
        note: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
        evidenceReferences: [{
          evidenceName: 'RNA-seq using two enrichment strategies for primary transcripts and consistent biological replicates',
          evidenceCode: 'RS-EPT-CBR',
          evidenceType: 'Strong',
          referenceId: 23203884,
          referenceURL: 'https://www.ncbi.nlm.nih.gov/pubmed/23203884',
          referenceCitation: 'Heladia Salgado, Martin Peralta-Gil, Socorro Gama-Castro, Alberto Santos-Zavaleta, Luis Muñiz-Rascado, Jair S. García-Sotelo, Verena Weiss, Hilda Solano-Lira, Irma Martínez-Flores, Alejandra Medina-Rivera, Gerardo Salgado-Osorio, Shirley Alquicira-Hernández, Kevin Alquicira-Hernández, Alejandra López-Fuentes, Liliana Porrón-Sotelo, Araceli M. Huerta, César Bonavides-Martínez, Yalbi I. Balderas-Martínez, Lucia Pannier, Maricela Olvera, Aurora Labastida, Verónica Jiménez-Jacinto, Leticia Vega-Alvarado, Victor del Moral-Chávez, Alfredo Hernández-Alvarez, Enrique Morett, Julio Collado-Vides, RegulonDB v8.0: omics data sets, evolutionary conservation, regulatory phrases, cross-validated gold standards and more, Nucleic Acids Research, Volume 41, Issue D1, 1 January 2013, Pages D203–D213, https://doi.org/10.1093/nar/gks1201'
        }]
      }],
      statistics: {
        regulators: 3,
        regulatoryInteractions: 4,
        promoters: 3
      }
    },
    growthConditions: [{
      id: 'MCO000000245',
      controlCondition: 'Escherichia coli| metC knockout mutant| MOPS| S-nitrosoglutathione 50 µM; glucose| pH 7.0| 37.0 C| OD600 below 0.7',
      experimentalCondition: 'Escherichia coli| metC knockout mutant| MOPS| S-nitrosoglutathione 50 µM; glucose; homocysteine 1 mM| pH 7.0| 37.0 C| OD600 below 0.7',
      effect: 'repressed',
      evidenceReferences: [{
        evidenceName: 'Inferred from expression pattern',
        evidenceCode: 'IEP',
        evidenceType: 'Strong',
        referenceId: 8021175,
        referenceURL: 'https://www.ncbi.nlm.nih.gov/pubmed/8021175',
        referenceCitation: 'Characterization of the sigma 38-dependent expression of a core Escherichia coli starvation gene, pexB.O L Lomovskaya, J P Kidwell, A MatinJ Bacteriol. 1994 Jul; 176(13): 3928–3935.'
      }]
    }],
    organismName: 'E. coli K12'
  },
  regulon: {
    regulonProducts: [{
      regulatorId: 'RDBECOLIRG00001',
      name: 'Lon protease',
      sequence: 'MNPERSERIEIPVLPLRDVVVYPHMVIPLFVGREKSIRCLEAAMDHDKKIMLVAQKEASTDEPGVNDLFTVGTVASILQMLKLPDGTVKVLVEGLQRARISALSDNGEHFSAKAEYLESPTIDEREQEVLVRTAISQFEGYIKLNKKIPPEVLTSLNSIDDPARLADTIAAHMPLKLADKQSVLEMSDVNERLEYLMAMMESEIDLLQVEKRIRNRVKKQMEKSQREYYLNEQMKAIQKELGEMDDAPDENEALKRKIDAAKMPKEAKEKAEAELQKLKMMSPMSAEATVVRGYIDWMVQVPWNARSKVKKDLRQAQEILDTDHYGLERVKDRILEYLAVQSRVNKIKGPILCLVGPPGVGKTSLGQSIAKATGRKYVRMALGGVRDEAEIRGHRRTYIGSMPGKLIQKMAKVGVKNPLFLLDEIDKMSSDMRGDPASALLEVLDPEQNVAFSDHYLEVDYDLSDVMFVATSNSMNIPAPLLDRMEVIRLSGYTEDEKLNIAKRHLLPKQIERNALKKGELTVDDSAIIGIIRYYTREAGVRGLEREISKLCRKAVKQLLLDKSLKHIEINGDNLHDYLGVQRFDYGRADNENRVGQVTGLAWTEVGGDLLTIETACVPGKGKLTYTGSLGEVMQESIQAALTVVRARAEKLGINPDFYEKRDIHVHVPEGATPKDGPSAGIAMCTALVSCLTGNPVRADVAMTGEITLRGQVLPIGGLKEKLLAAHRGGIKTVLIPFENKRDLEEIPDNVIADLDIHPVKRIEEVLTLALQNEPSGMQVVTAK',
      molecularWeight: 87.438,
      isoelectricPoint: 6.313,
      cellularLocation: 'cytosol',
      synonyms: ['Muc', 'Dir', 'Deg', 'LopA', 'CapR', 'Lon', 'DNA-binding, ATP-dependent protease La'],
      motifs: [{
        sequence: 'M',
        type: '|Cleavage-of-Initial-Methionine|',
        note: 'UniProt: Removed.'
      }, {
        leftEndPosition: 570,
        rightEndPosition: 772,
        sequence: 'INGDNLHDYLGVQRFDYGRADNENRVGQVTGLAWTEVGGDLLTIETACVPGKGKLTYTGSLGEVMQESIQAALTVVRARAEKLGINPDFYEKRDIHVHVPEGATPKDGPSAGIAMCTALVSCLTGNPVRADVAMTGEITLRGQVLPIGGLKEKLLAAHRGGIKTVLIPFENKRDLEEIPDNVIADLDIHPVKRIEEVLTLALQ',
        type: '|Pfam-Features|'
      }, {
        leftEndPosition: 264,
        rightEndPosition: 317,
        sequence: 'PKEAKEKAEAELQKLKMMSPMSAEATVVRGYIDWMVQVPWNARSKVKKDLRQAQ',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 5; AAA24078).'
      }, {
        leftEndPosition: 356,
        rightEndPosition: 363,
        sequence: 'GPPGVGKT',
        type: '|Nucleotide-Phosphate-Binding-Regions|',
        note: 'UniProt: ATP.'
      }, {
        leftEndPosition: 240,
        rightEndPosition: 252,
        sequence: 'ELGEMDDAPDENE',
        type: '|Protein-Segments|',
        note: 'UniProt: Asp/Glu-rich (acidic); Sequence Annotation Type: compositionally biased region.'
      }, {
        sequence: 'M',
        type: '|Active-Sites|'
      }, {
        leftEndPosition: 10,
        rightEndPosition: 201,
        sequence: 'EIPVLPLRDVVVYPHMVIPLFVGREKSIRCLEAAMDHDKKIMLVAQKEASTDEPGVNDLFTVGTVASILQMLKLPDGTVKVLVEGLQRARISALSDNGEHFSAKAEYLESPTIDEREQEVLVRTAISQFEGYIKLNKKIPPEVLTSLNSIDDPARLADTIAAHMPLKLADKQSVLEMSDVNERLEYLMAMME',
        type: '|Pfam-Features|'
      }, {
        leftEndPosition: 679,
        rightEndPosition: 679,
        sequence: 'S',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity.'
      }, {
        leftEndPosition: 772,
        rightEndPosition: 772,
        sequence: 'Q',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 4; AAA16837).'
      }, {
        leftEndPosition: 307,
        rightEndPosition: 307,
        sequence: 'S',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 4; AAA16837).'
      }, {
        leftEndPosition: 2,
        rightEndPosition: 784,
        sequence: 'NPERSERIEIPVLPLRDVVVYPHMVIPLFVGREKSIRCLEAAMDHDKKIMLVAQKEASTDEPGVNDLFTVGTVASILQMLKLPDGTVKVLVEGLQRARISALSDNGEHFSAKAEYLESPTIDEREQEVLVRTAISQFEGYIKLNKKIPPEVLTSLNSIDDPARLADTIAAHMPLKLADKQSVLEMSDVNERLEYLMAMMESEIDLLQVEKRIRNRVKKQMEKSQREYYLNEQMKAIQKELGEMDDAPDENEALKRKIDAAKMPKEAKEKAEAELQKLKMMSPMSAEATVVRGYIDWMVQVPWNARSKVKKDLRQAQEILDTDHYGLERVKDRILEYLAVQSRVNKIKGPILCLVGPPGVGKTSLGQSIAKATGRKYVRMALGGVRDEAEIRGHRRTYIGSMPGKLIQKMAKVGVKNPLFLLDEIDKMSSDMRGDPASALLEVLDPEQNVAFSDHYLEVDYDLSDVMFVATSNSMNIPAPLLDRMEVIRLSGYTEDEKLNIAKRHLLPKQIERNALKKGELTVDDSAIIGIIRYYTREAGVRGLEREISKLCRKAVKQLLLDKSLKHIEINGDNLHDYLGVQRFDYGRADNENRVGQVTGLAWTEVGGDLLTIETACVPGKGKLTYTGSLGEVMQESIQAALTVVRARAEKLGINPDFYEKRDIHVHVPEGATPKDGPSAGIAMCTALVSCLTGNPVRADVAMTGEITLRGQVLPIGGLKEKLLAAHRGGIKTVLIPFENKRDLEEIPDNVIADLDIHPVKRIEEVLTLALQNEPSGMQVVTAK',
        type: '|Chains|',
        note: 'UniProt: Lon protease.'
      }, {
        leftEndPosition: 255,
        rightEndPosition: 270,
        sequence: 'KRKIDAAKMPKEAKEK',
        type: '|Protein-Segments|',
        note: 'UniProt: Arg/Lys-rich (basic); Sequence Annotation Type: compositionally biased region.'
      }, {
        leftEndPosition: 11,
        rightEndPosition: 202,
        sequence: 'IPVLPLRDVVVYPHMVIPLFVGREKSIRCLEAAMDHDKKIMLVAQKEASTDEPGVNDLFTVGTVASILQMLKLPDGTVKVLVEGLQRARISALSDNGEHFSAKAEYLESPTIDEREQEVLVRTAISQFEGYIKLNKKIPPEVLTSLNSIDDPARLADTIAAHMPLKLADKQSVLEMSDVNERLEYLMAMMES',
        type: '|Conserved-Regions|',
        note: 'UniProt: Lon N-terminal.'
      }, {
        sequence: 'M',
        type: '|Active-Sites|'
      }, {
        leftEndPosition: 362,
        rightEndPosition: 362,
        sequence: 'K',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity and ATP-binding. No increased persister cell formation.'
      }, {
        leftEndPosition: 665,
        rightEndPosition: 665,
        sequence: 'H',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity.'
      }, {
        leftEndPosition: 743,
        rightEndPosition: 743,
        sequence: 'D',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: No effect.'
      }, {
        leftEndPosition: 352,
        rightEndPosition: 489,
        sequence: 'LCLVGPPGVGKTSLGQSIAKATGRKYVRMALGGVRDEAEIRGHRRTYIGSMPGKLIQKMAKVGVKNPLFLLDEIDKMSSDMRGDPASALLEVLDPEQNVAFSDHYLEVDYDLSDVMFVATSNSMNIPAPLLDRMEVIR',
        type: '|Pfam-Features|'
      }, {
        leftEndPosition: 211,
        rightEndPosition: 219,
        sequence: 'KRIRNRVKK',
        type: '|Protein-Segments|',
        note: 'UniProt: Arg/Lys-rich (basic); Sequence Annotation Type: compositionally biased region.'
      }, {
        leftEndPosition: 676,
        rightEndPosition: 676,
        sequence: 'D',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity.'
      }, {
        leftEndPosition: 779,
        rightEndPosition: 784,
        sequence: 'QVVTAK',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 2; AAA24079).'
      }, {
        leftEndPosition: 539,
        rightEndPosition: 563,
        sequence: 'AGVRGLEREISKLCRKAVKQLLLDK',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 5; AAA24078).'
      }, {
        leftEndPosition: 273,
        rightEndPosition: 273,
        sequence: 'A',
        type: '|Sequence-Conflicts|',
        note: 'UniProt: (in Ref. 4; AAA16837).'
      }, {
        leftEndPosition: 592,
        rightEndPosition: 773,
        sequence: 'ENRVGQVTGLAWTEVGGDLLTIETACVPGKGKLTYTGSLGEVMQESIQAALTVVRARAEKLGINPDFYEKRDIHVHVPEGATPKDGPSAGIAMCTALVSCLTGNPVRADVAMTGEITLRGQVLPIGGLKEKLLAAHRGGIKTVLIPFENKRDLEEIPDNVIADLDIHPVKRIEEVLTLALQN',
        type: '|Conserved-Regions|',
        note: 'UniProt: Lon proteolytic.'
      }, {
        leftEndPosition: 362,
        rightEndPosition: 362,
        sequence: 'K',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity and ATP-binding.'
      }, {
        leftEndPosition: 667,
        rightEndPosition: 667,
        sequence: 'H',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity.'
      }, {
        leftEndPosition: 679,
        rightEndPosition: 679,
        sequence: 'S',
        type: '|Mutagenesis-Variants|',
        note: 'UniProt: Loss of proteolytic activity. No increased persister cell formation.'
      }],
      externalCrossReferences: [{
        id: 'PRO_000022617',
        name: 'Protein Ontology',
        url: 'http://pir.georgetown.edu/cgi-bin/pro/entry_pro?id=PRO_000022617'
      }, {
        id: 'NP_418732',
        name: 'NCBI Reference Sequences Database',
        url: 'http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=NP_418732'
      }, {
        id: 'DIP-47842N',
        name: 'Database of Interacting Proteins',
        url: 'http://dip.doe-mbi.ucla.edu/dip/Browse.cgi?ID=DIP-47842N'
      }, {
        id: 'PF00589',
        name: 'Pfam',
        url: 'http://pfam.xfam.org/family/PF00589'
      }, {
        id: 'P0ADH5',
        name: 'Pride',
        url: ' http://www.ebi.ac.uk/pride/searchSummary.do?queryTypeSelected=identification%20accession%20number&identificationAccessionNumber=P0ADH5'
      }, {
        id: 'P0ADH5',
        name: 'MODBASE',
        url: 'http://modbase.compbio.ucsf.edu/modbase-cgi/model_search.cgi?searchkw=name&kword=P0ADH5'
      }, {
        id: 'IPR011010',
        name: 'InterPro',
        url: 'http://www.ebi.ac.uk/interpro/entry/IPR011010'
      }, {
        id: 'MINT-1221137',
        name: 'Mint',
        url: 'http://mint.bio.uniroma2.it/mint/search/search.do?queryType=protein&interactorAc=MINT-1221137'
      }, {
        id: 'P0ADH5',
        name: 'SWISS-MODEL',
        url: 'http://swissmodel.expasy.org/repository/smr.php?sptr_ac=P0ADH5'
      }, {
        id: 'P0ADH5',
        name: 'UniProt',
        url: 'https://www.uniprot.org/uniprot/P0ADH5'
      }, {
        id: 'b4312',
        name: 'EcoliWiki',
        url: 'http://ecoliwiki.net/colipedia/index.php/b4312'
      }, {
        id: 'P0ADH5',
        name: 'Protein Model Portal',
        url: 'http://www.proteinmodelportal.org/query/uniprot/P0ADH5'
      }],
      evidenceReferences: [{
        evidenceName: 'Assay of unpurified protein ',
        evidenceCode: 'AUP',
        evidenceType: 'Strong',
        referenceId: 8878036,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/8878036',
        referenceCitation: 'Gally, D.L., Leathart, J. and Blomfield, I.C. (1996), Interaction of FimB and FimE with the fim switch that controls the phase variation of type 1 fimbriae in Escherichia coli K‐12. Molecular Microbiology, 21: 725-738. doi:10.1046/j.1365-2958.1996.311388.x'
      }, {
        referenceId: 10648536,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/10648536',
        referenceCitation: 'Antigen 43 and Type 1 Fimbriae Determine Colony Morphology of Escherichia coli K-12 Henrik Hasman, Mark A. Schembri, Per Klemm Journal of Bacteriology Feb 2000, 182 (4) 1089-1095; DOI: 10.1128/JB.182.4.1089-1095.2000'
      }, {
        referenceId: 11881836,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/11881836',
        referenceCitation: 'Denise M. Wolf and Adam P. Arkin.OMICS: A Journal of Integrative Biology.Jan 2002.91-114.'
      }, {
        referenceId: 10781567,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/10781567',
        referenceCitation: 'Interaction of the FimB Integrase with thefimS Invertible DNA Element in Escherichia coli In Vivo and In Vitro Lesley S. Burns, Stephen G. J. Smith, Charles J. Dorman Journal of Bacteriology May 2000, 182 (10) 2953-2959; DOI: 10.1128/JB.182.10.2953-2959.2000'
      }, {
        referenceId: 16988236,
        referenceURL: 'http://www.ncbi.nlm.nih.gov/pubmed/16988236',
        referenceCitation: 'Teng CH, Xie Y, Shin S, et al. Effects of ompA deletion on expression of type 1 fimbriae in Escherichia coli K1 strain RS218 and on the association of E. coli with human brain microvascular endothelial cells. Infect Immun. 2006;74(10):5609–5616. doi:10.1128/IAI.00321-06'
      }]
    }]
  }
};
module.exports = {
  geneResponse
};